src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -580,6 +583,26 @@ public function testSubmitMultipleExpandedWithNegativeIntegerId()
         $this->assertFalse($field['2']->getData());
     }
 
+    public function testSubmitAssociationKey()
+    {
+        $single = new SingleIntIdEntity(1, 'Foo');
+        $assoc = new AssociationKeyEntity($single);
+
+        $this->persist(array($single, $assoc));
+
+        $field = $this->factory->createNamed('name', 'entity', null, array(
```

Comment:
hm, actually that only needs to be changed in 2.8 where this has been deprecated. So the merger would need to make the changes.